examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -7646,12 +7649,13 @@ endpoint 2 {
     callback attribute serverList;
     callback attribute clientList;
     callback attribute partsList;
+    callback attribute tagList;
     callback attribute generatedCommandList;
     callback attribute acceptedCommandList;
     callback attribute eventList;
     callback attribute attributeList;
-    ram      attribute featureMap default = 0;
-    callback attribute clusterRevision default = 1;
+    ram      attribute featureMap default = 1;
```

Comment:
This makes it seem like the featuremap value for descriptor comes from ZAP. It does not. It is auto-computed based on whether the taglist is present. Suggest leaving this as 0.

Reply:
Sure.